src/main/java/fact/utils/CombineDataArrays.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * 
+ */
+package fact.utils;
+
+import fact.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+
+/**
+ * Subtract two Pixel Arrays of the same size from each other
+ * 
+ * @author  Michael Bulinski &lt;michael.bulinski@udo.edu&gt;
+ *
+ */
+public class CombineDataArrays implements Processor {
+	static Logger log = LoggerFactory.getLogger(RemappingKeys.class);
+	
+    @Parameter(required = true, description = "The key to your first array.")
+    private String firstArrayKey;
+    @Parameter(required = true, description = "The key to your second array.")
+    private String secondArrayKey;
+    @Parameter(required = false, description = "The key for the resulting array.")
+    private String outputKey;
+    @Parameter(required = true, description = "The operation to perform, (add, sub, mul, div)")
+	private String op;
+    
+	private int npix = 1440*300;
+    
+
+	/* (non-Javadoc)
+	 * @see stream.Processor#process(stream.Data)
+	 */
+	@Override
+	public Data process(Data input) {
+		Utils.isKeyValid(input, firstArrayKey, double[].class);
+		Utils.isKeyValid(input, secondArrayKey, double[].class);
+
+		double[] resultArray =  new double[npix];
+
+		double[] array1 	 = (double[]) input.get(firstArrayKey);
+		double[] array2 	 = (double[]) input.get(secondArrayKey);
+
+		if (op.equals("add")) {
+			for (int pix = 0; pix < npix; pix++) {
```

Comment:
instead of using npix you could make this processor generic looping over array elements, but then you need to have a sanity check if both arrays have the same length. But this is allready implemented in `fact.photonstream.timeSeriesExtraction.ElementWise`